### PR TITLE
fix: Linter fixes for plugins/inputs/[t-z]*

### PR DESCRIPTION
**Pull request by zak-pawel**
Following findings were fixed:
```
plugins/inputs/tail/multiline.go:125:3             revive       bare-return: avoid using bare returns, please add return expressions
plugins/inputs/tail/multiline.go:129:3             revive       bare-return: avoid using bare returns, please add return expressions
plugins/inputs/tail/tail_test.go:502:37            revive       empty-lines: extra empty line at the start of a block
plugins/inputs/tcp_listener/tcp_listener.go:6:2    revive       imports-blacklist: should not use the following blacklisted import: "log"
plugins/inputs/twemproxy/twemproxy.go:127:14       revive       unnecessary-stmt: switch with only one case can be replaced by an if-then
plugins/inputs/twemproxy/twemproxy_test.go:76:4    staticcheck  SA4004: the surrounding loop is unconditionally terminated
plugins/inputs/udp_listener/udp_listener.go:5:2    revive       imports-blacklist: should not use the following blacklisted import: "log"
plugins/inputs/udp_listener/udp_listener.go:175:5  revive       early-return: if c { } else {... return} can be simplified to if !c { ... return }
plugins/inputs/uwsgi/uwsgi.go:81:56                revive       import-shadowing: The name 'url' shadows an import name
plugins/inputs/uwsgi/uwsgi.go:103:28               bodyclose    response body must be closed
plugins/inputs/vsphere/endpoint.go:28:14           gosimple     S1007: should use raw string (`...`) with regexp.MustCompile to avoid having to escape twice
plugins/inputs/vsphere/endpoint.go:118:56          revive       import-shadowing: The name 'url' shadows an import name
plugins/inputs/vsphere/endpoint.go:587:55          revive       import-shadowing: The name 'filter' shadows an import name
plugins/inputs/vsphere/endpoint.go:608:52          revive       import-shadowing: The name 'filter' shadows an import name
plugins/inputs/vsphere/endpoint.go:660:49          revive       import-shadowing: The name 'filter' shadows an import name
plugins/inputs/vsphere/endpoint.go:678:47          revive       import-shadowing: The name 'filter' shadows an import name
plugins/inputs/vsphere/endpoint.go:768:54          revive       import-shadowing: The name 'filter' shadows an import name
plugins/inputs/vsphere/endpoint.go:891:9           revive       import-shadowing: The name 'object' shadows an import name
plugins/inputs/vsphere/endpoint.go:1275:1          revive       confusing-results: unnamed results of the same type may be confusing, consider using named results
plugins/inputs/vsphere/finder.go:66:18             revive       confusing-naming: Method 'find' differs only by capitalization to method 'Find' in the same source file
plugins/inputs/vsphere/finder.go:110:2             revive       import-shadowing: The name 'types' shadows an import name
plugins/inputs/vsphere/tscache.go:4:2              revive       imports-blacklist: should not use the following blacklisted import: "log"
plugins/inputs/vsphere/tscache.go:59:54            revive       import-shadowing: The name 'time' shadows an import name
plugins/inputs/vsphere/vsphere_test.go:231:2       revive       empty-lines: extra empty line at the end of a block
plugins/inputs/vsphere/vsphere_test.go:315:5       ineffassign  ineffectual assignment to err
plugins/inputs/vsphere/vsphere_test.go:431:5       ineffassign  ineffectual assignment to err
plugins/inputs/vsphere/vsphere_test.go:460:6       revive       confusing-naming: Method 'testCollection' differs only by capitalization to function 'TestCollection' in the same source file
plugins/inputs/wireguard/wireguard.go:5:2          revive       imports-blacklist: should not use the following blacklisted import: "log"
plugins/inputs/wireless/wireless_linux.go:8:2      revive       imports-blacklist: should not use the following blacklisted import: "log"
plugins/inputs/wireless/wireless_linux.go:131:16   revive       import-shadowing: The name 'path' shadows an import name
plugins/inputs/zipkin/zipkin.go:112:3              staticcheck  SA2000: should call wg.Add(1) before starting the goroutine to avoid a race
plugins/inputs/zipkin/zipkin_test.go:652:20        bodyclose    response body must be closed
```

### CODE REVIEW

File: plugins/inputs/wireless/wireless_test.go [hunk 1]
```diff
@@ -43,11 +45,15 @@ func TestLoadWirelessTable(t *testing.T) {
 			Beacon:    int64(0),
 		},
 	}
-	metrics, err := loadWirelessTable(testInput)
+
+	w := Wireless{
+		Log: testutil.Logger{},
+	}
+	metrics, err := w.loadWirelessTable(testInput)
```
Comment (srebhan) [inline plugins/inputs/wireless/wireless_test.go:52]: The follow-up lines could be replaced by `require.NoError()`...

File: plugins/inputs/x509_cert/x509_cert_test.go [hunk 2]
```diff
@@ -199,30 +197,30 @@ func TestTags(t *testing.T) {
...
-	assert.True(t, acc.HasTag("x509_cert", "san"))
-	assert.Equal(t, "localhost,127.0.0.1", acc.TagValue("x509_cert", "san"))
+	require.True(t, acc.HasTag("x509_cert", "san"))
+	require.Equal(t, "localhost,127.0.0.1", acc.TagValue("x509_cert", "san"))
 
-	assert.True(t, acc.HasTag("x509_cert", "serial_number"))
+	require.True(t, acc.HasTag("x509_cert", "serial_number"))
 	serialNumber := new(big.Int)
 	_, validSerialNumber := serialNumber.SetString(acc.TagValue("x509_cert", "serial_number"), 16)
 	if !validSerialNumber {
```
Comment (srebhan) [inline plugins/inputs/x509_cert/x509_cert_test.go:220]: How about `require.Truef()`?

**Review by srebhan — COMMENTED** [review]
@zak-pawel thanks for your amazing effort on this! Much appreciated! I have two suggestions for improvement, but please feel free to ignore them as they are not in the current PR.

File: plugins/inputs/wireless/wireless_test.go (hunk 1 above)
Comment (zak-pawel) [reply plugins/inputs/wireless/wireless_test.go:52]: Done

File: plugins/inputs/x509_cert/x509_cert_test.go (hunk 2 above)
Comment (zak-pawel) [reply plugins/inputs/x509_cert/x509_cert_test.go:220]: I don't if I want to go that deep with changing all `True`'s to `Truef`'s...

**Review by powersj — APPROVED** [review]

File: plugins/inputs/x509_cert/x509_cert_test.go (hunk 2 above)
Comment (srebhan) [reply plugins/inputs/x509_cert/x509_cert_test.go:220]: I think my comment was misleading. I meant the
```golang
	_, validSerialNumber := serialNumber.SetString(acc.TagValue("x509_cert", "serial_number"), 16)
	if !validSerialNumber {
		t.Errorf("Expected a valid Hex serial number but got %s", acc.TagValue("x509_cert", "serial_number"))
	}
``` 
part... :-)

Comment (zak-pawel) [reply plugins/inputs/x509_cert/x509_cert_test.go]: Makes more sense, changed :)

**Review by srebhan — APPROVED** [review]
Perfect. Thanks @zak-pawel!

**Review by powersj — APPROVED** [review]

**Pull request merged**